src/util/antlr/ANTLRErrorHandling.h

```diff
@@ -11,51 +11,26 @@
 #include <string>
 
 #include "BaseErrorListener.h"
-#include "CharStream.h"
-#include "CommonTokenStream.h"
-#include "DefaultErrorStrategy.h"
 #include "ParserRuleContext.h"
-#include "RecognitionException.h"
 #include "Recognizer.h"
-#include "TokenSource.h"
+#include "Token.h"
+#include "absl/strings/str_cat.h"
 
-// TODO<qup42> whats up with this namespace?
-namespace {
 ExceptionMetadata generateMetadata(antlr4::Recognizer* recognizer,
-                                   antlr4::Token* offendingToken,
-                                   [[maybe_unused]] size_t line,
-                                   [[maybe_unused]] size_t charPositionInLine) {
-  // TODO<qup42> the source query differs from the real query. The SparqlLexer
-  //  lowercases some parts and sometimes adds whitespaces.
-  return {((antlr4::CommonTokenStream*)recognizer->getInputStream())
-              ->getTokenSource()
-              ->getInputStream()
-              ->toString(),
-          offendingToken->getStartIndex(), offendingToken->getStopIndex()};
-}
+                                   antlr4::Token* offendingToken, size_t line,
+                                   size_t charPositionInLine);
 
 [[maybe_unused]] ExceptionMetadata generateMetadata(
-    antlr4::ParserRuleContext* ctx) {
-  // TODO<qup42> see above
-  return {ctx->getStart()->getInputStream()->toString(),
-          ctx->getStart()->getStartIndex(), ctx->getStop()->getStopIndex()};
-}
-}  // namespace
+    antlr4::ParserRuleContext* ctx);
```

Comment:
What is the point of the `[[maybe_unused]]` here?

Reply:
Good spot! This function was unused for some time and I forgot to remove that.

Reply:
Alright, then as soon as either the `[[maybe_unused]]` or the function (if unused) are removed, I am going to merge this.